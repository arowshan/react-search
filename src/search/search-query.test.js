import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import SearchQuery from './search-query';
import renderer from 'react-test-renderer';


describe('Components rendered', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SearchQuery />);
    return wrapper;
  });

  test('has an input', () => {
    expect(wrapper.find('input').length).toBe(1);
  });

  test('has a button with search-button class', () => {
    expect(wrapper.find('.search-button').length).toBe(1);
  });

  test("should render form", () => {
    expect(wrapper.find("form")).toMatchSnapshot();
  });

  // test("should render input", () => {
  //   expect(wrapper.find("input")).toMatchSnapshot();
  // });
  
});
