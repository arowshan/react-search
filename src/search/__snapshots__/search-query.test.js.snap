// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components rendered should render form 1`] = `
<form
  className="query-container"
  onSubmit={[Function]}
>
  <input
    className="search-input"
    name="searchInput"
    onBlur={[Function]}
    onFocus={[Function]}
    placeholder="Search..."
  />
  <RaisedButton
    className="search-button"
    disabled={false}
    fullWidth={false}
    icon={<ActionSearch />}
    labelPosition="after"
    primary={true}
    secondary={false}
  />
</form>
`;
